.travis.yml

```diff
@@ -12,7 +12,6 @@ addons:
 node_js:
   - "8"
   - "6"
```

Comment:
Does it actually fail on 4 - or just not supported on 4 ? (ie have they actively broken it ?)

Reply:
It's actually broken